content/docs/projects/approver-policy/README.md

```diff
@@ -415,7 +415,9 @@ spec:
         val-1: key-1
 ```
 
-There are currently no open source plugins.
+There is currently only one (experimental) open source plugin:
```

Comment:
```suggestion
## Known Plugins
```

Could you also add a note at the end of this section to invite authors to share their plugins back to this page?

Reply:
Better now?